payments/card/seamless-view.md

```diff
@@ -361,7 +361,7 @@ The previous two steps gives this HTML:
         <script src=<YourJavaScriptFileHere>></script>
     </head>
     <body>
-        <div id="SweddbankPay-seamless-view-page">
+        <div id="SwedbankPay-seamless-view-page">
```

Comment:
This looks like the remnants of a search and replace. For consistency, the `id` value should be `all-lower-kebab-case`. Can you ensure that all HTML examples have an `id` value that matches this?

```suggestion
        <div id="swedbank-pay-seamless-view">
```